packages/fxa-settings/src/lib/crypto/scoped-keys.ts

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// NOTE: Ported from scoped-keys.js in content server. Minor typescript adaptions applied.
+
+// TOOD: Why can't this be imported?
+const fxaCryptoDeriver = require('fxa-crypto-relier/dist/fxa-crypto-relier/fxa-crypto-deriver');
```

Comment:
Hmm maybe it doesn't use modules? I didn't know we even had this repo around, there's a lot of dependabots over there. If we're the only ones that use it, I wonder if it's worth porting over directly into our codebase at some point, it doesn't look _that_ heavy...

Reply:
Yeah, definitely worth a follow up imo.